docs/gl_objects/status_checks.rst

```diff
@@ -0,0 +1,57 @@
+############
+Status Checks
+############
```

Comment:
Could we align the naming with the upstream documentation and the endpoints? This way if there later some internal checks that GitLab implements, we don't have a collision.

```suggestion
#################
External Status Checks
#################
```